linux_os/guide/system/permissions/files/permissions_local_var_log/bash/ubuntu.sh

```diff
@@ -0,0 +1,14 @@
+# platform = multi_platform_ubuntu
+
+readarray -t files < <(find /var/log/)
```

Comment:
This is not robust, consider using this approach: https://github.com/koalaman/shellcheck/wiki/SC2044

Reply:
readarray is extensively used in CaC's code. Maybe we should leave as is in this PR and fix it altogether in a separate PR.
Thoughts?

Reply:
Our file owner, groupowner, permissions template is already using this approach. I'm fine with this approach for now and then we can fix everything related to this in a different PR.

Reply:
`readarray` in itself is fine, it's just the combination of `find` and `readarray` that is problematic. I am fine to address it later, #8032 will remind us :-)